.gitignore

```diff
@@ -1,3 +1,10 @@
+	## this should ignore all files in the pipeline ##
+	
+data/input/*
+data/out/*/*.tsv
+data/out/*/*.fasta
+data/out/*/*.new
+
```

Comment:
I think it would be good to add `*.png`, and maybe `*.pdf` as well, since some of these files ended up getting committed as well.

Files can sneak into repositories when you do things like `git add .` followed by `git commit`. It is good practice to instead use `git add path/to/file` to each file individually, and then do `git status` before running `git commit`. This way, you know exactly what is being added.